doc/md/integrations/circleci-orbs.mdx

```diff
@@ -0,0 +1,544 @@
+          ACCEPT_EULA: Y
+          MSSQL_PID: Developer
+          MSSQL_SA_PASSWORD: P@ssw0rd0995
+    environment:
+      # Define those envs in CircleCI context to keep them secure
+      ATLAS-TOKEN: "your-atlas-token"
+      GITHUB_TOKEN: "your-github-token"
+      GITHUB_REPOSITORY: "your-repo-name" # for example, "ariga/atlas-orb"
+    steps:
+      - checkout
+      - run:
+          name: Wait for SQL Server
+          command: dockerize -wait tcp://127.0.0.1:1433 -timeout 60s
+      - atlas-orb/setup:
+          version: "latest"
+      - atlas-orb/migrate_lint:
+          dir_name: my-project
+          dev_url: "sqlserver://sa:P@ssw0rd0995@localhost:1433?database=master"
+workflows:
+  use-atlas:
+    jobs:
+      - push-dir
+```
+
+</TabItem>
+<TabItem value="clickhouse">
+
+```yaml
+version: 2.1
+orbs:
+  atlas-orb: ariga/atlas-orb@0.0.6
+jobs:
+  push-dir:
+    docker:
+      - image: cimg/base:current
+      - image: clickhouse/clickhouse-server:23.10
+        environment:
+          CLICKHOUSE_DB: test
+          CLICKHOUSE_DEFAULT_ACCESS_MANAGEMENT: 1
+          CLICKHOUSE_PASSWORD: pass
+          CLICKHOUSE_USER: root
+    environment:
+      # Define those envs in CircleCI context to keep them secure
+      ATLAS-TOKEN: "your-atlas-token"
+      GITHUB_TOKEN: "your-github-token"
+      GITHUB_REPOSITORY: "your-repo-name" # for example, "ariga/atlas-orb"
+    steps:
+      - checkout
+      - run:
+          name: Wait for Clickhouse
+          command: dockerize -wait tcp://127.0.0.1:9000 -timeout 60s
+      - atlas-orb/setup:
+          version: "latest"
+      - atlas-orb/migrate_lint:
+          dir_name: my-project
+          dev_url: "clickhouse://root:pass@localhost:9000/test"
+workflows:
+  use-atlas:
+    jobs:
+      - push-dir
+```
+
+</TabItem>
+</Tabs>
+
+### Inputs
+
+All inputs are optional as they may be specified in the Atlas configuration file.
+
+* `dir` - The URL of the migration directory to lint.  For example: `file://migrations`.
+  Read more about [Atlas URLs](https://atlasgo.io/concepts/url).
+* `dir_name` - The name (slug) of the project in Atlas Cloud.
+* `dev_url` - The URL of the dev-database to use for analysis.  For example: `mysql://root:pass@localhost:3306/dev`.
+  Read more about [dev-databases](https://atlasgo.io/concepts/dev-database).
+* `config` - The path to the Atlas configuration file.  By default, Atlas will look for a file
+  named `atlas.hcl` in the current directory. For example, `file://config/atlas.hcl`.
+  Learn more about [Atlas configuration files](https://atlasgo.io/atlas-schema/projects).
+* `env` - The environment to use from the Atlas configuration file.  For example, `dev`.
+* `github_token_env` (required) - The GitHub token to use for authentication. Defaults to `GITHUB_TOKEN` environment variable.
+* `github_repository_env` (required) - The GitHub repository to use for authentication. Defaults to `GITHUB_REPOSITORY` environment variable.
+
+:::warning
```

Comment:
This turns out as "DANGER" , which is a bit off-putting. Maybe info?

Reply:
And please explain what they are and what they are used for

Reply:
I changed the info and also edited the description of `github_repository_env` to more clear
```
* `github_token_env` (required) - The GitHub token to use for authentication.
  Defaults to `GITHUB_TOKEN` environment variable.
* `github_repository_env` (required) - The GitHub repository name that will be used for reporting lint results.
  Defaults to `GITHUB_REPOSITORY` environment variable.
```